packages/output/src/browser/output-widget.tsx

```diff
@@ -27,8 +28,8 @@ export const OUTPUT_WIDGET_KIND = 'outputView';
 @injectable()
 export class OutputWidget extends ReactWidget {
 
-    @inject(OutputChannelManager)
-    protected readonly outputChannelManager: OutputChannelManager;
```

Comment:
I think `OutputChannelManager` is better name for a facade service managing all output channels then `OutputChannelReaders`. I would try to revert this file and find better naming.

Reply:
I wanted the name to make it clear that this class was intended to be used only by the output view or anything else that may show output channels, not to be used to write to output channels.  I.e. it provides read-only access to the channels.

OutputChannelManager is the injection used to get channels for writing to.  We certainly can't change that, so are you suggesting that we re-combine OutputChannelManager and OutputChannelReaders into a single class?  IMO pushing text into channels and extracting content from channels by a viewer are two different things that should be kept separate, and extenders get more flexibility when they are kept separate.